### DISCUSSION TITLE
Are SSM Sessions Logged in Journald?

### DISCUSSION THREAD

**Discussion by hi-artem** [bottlerocket-os/bottlerocket#2432] [Q&A] [answered]
Are sessions opened by ssm-agent suppose to be getting logged in journald? I would expect to see those under control-container logs, but they don't show up.

```sh
bash-5.1# journalctl -u  host-containers@control.service
-- No entries --
```

**Comment by jpculp** [comment, marked as answer]
Thanks for reaching out, @hi-artem! They are not logged on the host side today. You can see the sessions by querying Systems Manager itself either through the web console or CLI (using `aws ssm describe-sessions --state History`). You can also configure Systems Manager to log sessions to [CloudWatch or S3](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-logging.html). If you'd still like to see this on the host itself, please let us know and we can create a feature request for you.

**Reply by hi-artem** [reply]
@jpculp Thank you for answering the question. 

Tracking sessions based on Session Manager events works for our particular use-case. No need to open a feature request.

**Comment by James-Quigley** [comment]
Is this still the case? Are SSM logs stored anywhere on the host? (where a centralized log system might be able to scrape them?) or do we have to rely on cloudwatch/s3?

**Reply by vigh-m** [reply]
Hi! Thanks for reaching out.
I verified with the latest AMI release right now (v1.30.0) and this remains the case. Using `aws ssm describe-sessions --state History` and other AWS tooling is still the best way to get this information.

Could you share more details about your use case? We're happy to open a Feature Request if needed.

**Comment by James-Quigley** [comment]
We have a daemonset on every node that collects pod logs and some node logs for our centralized log pipeline (Vector + Loki). We’d like to have it be able to ingest logs from SSM sessions started on the host. We’d can do this on AL2, but not bottle rocket
